.github/workflows/develop_update.yml

```
@@ -0,0 +1,31 @@
+name: Update develop after master merge
+
+on:
+  push:
+    branches:
+    - master
+
+
+jobs:
+  develop_update:
+    runs-on: ubuntu-latest
+
+    if: github.repository == 'qmk/qmk_firmware'
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Update develop from master
+      shell: 'bash {0}'
+      run: |
+        if [ -e .locked ]; then
+            echo 'Develop branch is locked, aborting update!'
+            exit 1
+        fi
+
```

Something like this maybe?
```suggestion
    - name: Check development not locked
      run: |
        test-f .locked

    - name: Update develop from master
      run: |
```

---

I rearranged it into something that looks more like what you have. How's that look?